agents/loco_mc_agent.py

```diff
@@ -328,10 +322,13 @@ def perceive(self, force=False):
             # New chat, mark as unprocessed.
             self.memory.tag(subj_memid=chat_memid, tag_text="unprocessed")
             # Send data to the dashboard timeline
+            end_time = datetime.datetime.now()
             hook_data = {
                 "name" : "perceive",
+                "start_datetime" : start_time,
+                "end_datetime" : end_time,
                 "speaker" : speaker, 
-                "time" : self.last_chat_time,
+                "agent_time" : self.last_chat_time,
```

Comment:
it's not the logical time the agent is using (line#314). You should use `self.get_time()` here.

Reply:
Thanks for catching this! I've fixed it in [232a38a](https://github.com/facebookresearch/droidlet/pull/526/commits/232a38a5cf2fec5c57535a1f46187c785ed97dd4).